content/applications/finance/accounting/bank/setup/outstanding_accounts.rst

```diff
@@ -0,0 +1,36 @@
+====================
+Outstanding Accounts
+====================
+
+In Odoo, before being recorded in your bank account, payments are registered through transitory
+accounts, named Outstanding Accounts.
+
+- The **Outstanding Payments Account** is where outgoing payments are posted until they are linked
+  to a withdrawal from your bank statement.
+- The **Outstanding Receipts Account** is where incoming payments are posted until they are linked
+  with a deposit from your bank statement.
+These accounts should be of type "Current Asset".
+
+.. note::
+   The movement from an Outstanding Account to a bank account is done automatically when you
+   reconcile the bank account with a bank statement.
+
+Configuration
+=============
+
+To configure your Default Outstanding Accounts from the Accounting app, go to the
+:menuselection:`Configuration --> Settings --> Default Accounts` and define your Outstanding
+Receipts or Payments Account.
+
+You can also decide to use specific Outstanding Accounts related to a specific journal.
+
+To do so, go to the :menuselection:`Dashboard --> Bank --> Configuration`. In the Incoming/Outgoing
+Payments tab, click on the dropdown column option to see and update your Outstanding accounts.
```

Comment:
```suggestion
To do so, go to your Dashboard, click the three vertical dots menu under Bank and click on Configuration. In the Incoming/Outgoing
Payments tab, click on the dropdown column option to see and update your Outstanding Accounts.
```

- It took me a few minutes to understand where I had to go with "Dashboard --> Bank --> Configuration". I know you don't want to use the word 'vertical ellipsis, but "three dots menu" or "three vertical dots menu" should do the trick and I understand using the menuselection directive with something like "Bank: three dots menu" might to be very clear (that's why I suggest putting that in text form)
- Issue: there is only a single Payments Configuration tab on v14. It seems it's where you need to set your default accounts. It's also quite different from v15.
- Capital A for "Outstanding Accounts" to be consistent